docs/Users_Guide/tc-diag.rst

```diff
@@ -94,12 +94,21 @@ Configuring input tracks and time
   valid_inc = [];
   valid_exc = [];
   valid_hour = [];
-  lead = [];
 
 The TC-Diag tool should be configured to filter the input track data (**-deck**) down to the subset of tracks that correspond to the gridded data files provided (**-data**). The filtered tracks should contain data for only *one initialization time* but may contain tracks for multiple models.
 
 The configuration options listed above are used to filter the input track data down to those that should be processed in the current run. These options are common to multiple MET tools and are described in :numref:`config_options_tc`.
 
+.. code-block:: none
```

Comment:
@JohnHalleyGotway There's something weird here. There's an empty code block, but then the lead times that follow are split across multiple lines in a weird way. Should these actually be inside the code block?

Reply:
Oh, I think I understand -- to be inside the code block, the lead = [ ]; stuff needs to be indented.

Reply:
@jvigh, great catch! Thanks for your review. I pushed a fix for that indent problem.